network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -636,14 +641,34 @@ def get_status_for_airtable(self):
             return "Live"
         return "Draft"
 
+    @property
+    def original_product(self):
+        return get_original_by_slug(ProductPage, self.slug)
+
+    def get_or_create_votes(self):
```

Comment:
this just got moved up, so that it's next to the other vote-related function/properties